intermediate-exercises/src/components/02-StopWatch.js

```diff
@@ -18,9 +18,63 @@ import React, { Component } from 'react';
 */
 
 class StopWatch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      running: false,
```

Comment:
As it has a boolean value, it can be named as 'isRunning'.

Reply:
Yeah Right, I will change it.